src/style.css

```
@@ -1 +1,45 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars-block {
+  flex-shrink: 0;
+  margin: 0;
+  padding: 0;
+}
+
+.stars{
+  display: flex;
+  margin: 0;
+}
+
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
```

just use absolute path
```suggestion
  background-image: url(images/star.svg);
```